aspnetcore/blazor/globalization-localization.md

```diff
@@ -58,7 +58,17 @@ Blazor WebAssembly apps set the culture using the user's [language preference](h
 
 To explicitly configure the culture, set <xref:System.Globalization.CultureInfo.DefaultThreadCurrentCulture?displayProperty=nameWithType> and <xref:System.Globalization.CultureInfo.DefaultThreadCurrentUICulture?displayProperty=nameWithType> in `Program.Main`.
 
-By default, Blazor's linker configuration for Blazor WebAssembly apps strips out internationalization information except for locales explicitly requested. For more information and guidance on controlling the linker's behavior, see <xref:blazor/host-and-deploy/configure-linker#configure-the-linker-for-internationalization>.
+::: moniker range=">= aspnetcore-5.0"
+
+By default, the trimming configuration for removing unused assemblies in Blazor WebAssembly apps strips out internationalization information except for locales explicitly requested. For more information, see <xref:blazor/host-and-deploy/configure-trimmer>.
```

Comment:
This isn't true any more. All culture information is included by default unless you configure `InvariantGlobalization`.